test/e2e/framework.go

```diff
@@ -1265,18 +1269,22 @@ func (data *TestData) deployFlowVisibility(config FlowVisibiltiySetUpConfig) (ch
 	return chSvc.Spec.ClusterIP, nil
 }
 
-func (data *TestData) deployFlowVisibilityCommon(yamlFile string) error {
+func (data *TestData) deployFlowVisibilityCommon(chOperatorYML, flowVisibilityYML string) error {
+	rc, _, _, err := data.provider.RunCommandOnNode(controlPlaneNodeName(), fmt.Sprintf("kubectl apply -f %s", chOperatorYML))
+	if err != nil || rc != 0 {
+		return fmt.Errorf("error when deploying the ClickHouse Operator YML; %s not available on the control-plane Node", chOperatorYML)
```

Comment:
I think maybe we can also include err in error message?

Reply:
done